File: src/2015/day-1/ts/part-1.ts
// Run: ts-node src/2015/day-1/ts/part-1.ts

import * as fs from "fs";

const input = fs.readFileSync("src/2015/day-1/input.txt", "utf-8");

const left = input.split("(").length - 1;
const right = input.split(")").length - 1;

console.log(`Floor #: ${left - right}`);

File: src/2015/1/python/part-1.py
# Run: python src/2015/1/python/part-1.py

import re

input_string = 'src/2015/1/input.txt'

with open(input_string, 'r') as input_file:
    for line in input_file:
        left = len(line.split('('))
        right = len(line.split(')'))

        print("Floor #: " + str(left - right))
